package.json

```diff
@@ -62,31 +62,31 @@
   },
   "dependencies": {
     "@google-cloud/common": "^0.18.2",
-    "async": "^2.0.1",
-    "extend": "^3.0.0",
+    "async": "^2.6.1",
+    "extend": "^3.0.1",
     "google-gax": "^0.16.0",
-    "google-proto-files": "^0.15.0",
-    "is": "^3.0.1",
-    "lodash.merge": "^4.6.0",
+    "google-proto-files": "^0.16.0",
+    "is": "^3.2.1",
+    "lodash.merge": "^4.6.1",
```

Comment:
aside:
* both `extend` and `loadash.merge` are needed and `Object.assign` doesn't work? (they are all different, but do our use-cases care?)

Reply:
Good question :)  `extend` was just not being used, and I drop it in #85.  `lodash.merge` is being used by the auto-generated code via gapic.  There are nuanced differences between `lodash.merge` and `Object.assign`, so that's something I would really prefer @alexander-fenster to chime in on :)